### DISCUSSION TITLE
Welcome to foundryvtt-starfinder Discussions!

### DISCUSSION THREAD

**Discussion by wildj79** [wildj79/foundryvtt-starfinder#348] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cerebrolator** [comment]
I don't know if this is the right place to ask or not.  If not, please let me know where I should ask.

I'm trying to figure out how to give NPCs spell-like abilities in the Starfinder system on Foundry VTT.  I know how to add spells but it doesn't let me set the caster level or the number of spells that can be cast.  There must be some way to indicate that an NPC has spell-like abilities or innate spells or something of that sort.  Do you know how I can find this information?

**Reply by wildj79** [reply]
Welcome!  

So, probably asking in a reply to the welcome message isn't the ideal place to ask. But, I will say that this is the type of question that is better suited for discord. You can find us in the official Foundry discord in the `#starfinder` channel.

https://discord.gg/foundryvtt

**Comment by cerebrolator** [comment]
Thank you!  I will definitely ask there.
